P2-Proposals/TeamPi/P2Proposal.md

```diff
@@ -0,0 +1,26 @@
+# Life in the Time of Corona
+
+## Description
+Our project will pull CDC data on coronavirus case counts by state, analyze and present the data to the user based on their input, create graphs and predictions of future case counts, and email the data to the user.
+
+## Automation
+1. Prompt user input:
+    * State/national data
+    * Graphs
+    * Predictions
+    * Email
+2. Download case count from CDC website
+3. Scrape data from downloaded file
+4. Data processing
+5. Generate graph with Python
+6. Generate ML prediction with Python
+7. Email user with case count data, graphs, predictions
+
+## Tech Stack
+* UiPath
+    * Excel activities
+    * Outlook activities
+* Python
+
+## Team Pi
```

Comment:
Please embed the flowchart in this file only.

Reply:
We will describe data processing in detail, I will embed the flowchart in our project proposal file!